src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,146 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+h1,
+h2,
+p {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  width: 198px;
+  height: 408px;
+}
+
+.card {
+  padding-top: 32px;
+  box-sizing: border-box;
```

Comment:
move to `*` selector and delete in all other places

Reply:
I didn't understand what to do here